package.json

```diff
@@ -20,7 +20,7 @@
       "ccdUrl": "http://ccd-data-store-api-demo.service.core-compute-demo.internal"
     },
     "aat": {
-      "cosUrl": "http://div-cos-staging.service.core-compute-aat.internal",
+      "cosUrl": "http://div-cos-pr-491.service.core-compute-preview.internal",
```

Comment:
REVERT THIS

Reply:
Can we revert this given it's tested please?

Reply:
Was used for other story (re-using PR) but have reverted